popup.js

```diff
@@ -0,0 +1,378 @@
+    descriptionpop1: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent`,
+    descriptionpop: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley 
+    of type and scrambled it 1960s. Lorem Ipsum is 
+    simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever 
+    since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of 
+    type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.`,
+    skills: ['HTML', 'css', 'javaScript', 'Ruby', 'Boostrap'],
+  },
+];
+
+const Workcards = document.getElementById('container');
+
+containersworks.forEach((item2) => {
+  Workcards.innerHTML += `
+<div class="${item2.class}">
+        <img
+          class="desktop-image"
+          src="${item2.desktop}"
+          alt="Project Image"
+        />
+        <img
+          class="snapshoot"
+          src="${item2.snapshoot}"
+          alt="Project Image"
+        />
+        <div class="daily">
+          <h3 class="Tonic">${item2.Tonic}</h3>
+          <ul class="offer">
+            <li class="canopy">${item2.offer[0]}</li>
+            <span class="point"></span>
+            <li class="backend">${item2.offer[1]}</li>
+            <span class="point"></span>
+            <li class="year">${item2.offer[2]}</li>
+          </ul>
+            <p class="description">${item2.description}</p>
+          <ul class="skills">
+            <li class="htl"><a href="#"></a>${item2.skills[0]}</li>
+            <li class="css"><a href="#"></a>${item2.skills[1]}</li>
+            <li class="js"><a href="#"></a>${item2.skills[2]}</li>
+          </ul>
+          <button class="seeproject" data-modal-target="#modal1">See Project</button>
+        </div>
+      </div>
+`;
+});
+
+const modal0 = document.querySelector('.modal0');
+
+modal0.insertAdjacentHTML(
+  'afterbegin',
+  `
+   <nav>
+        <h2 class='Tonicpop'>${containersworks[0].Tonic}</h2>
+        <button class='closepopup'>&times;</button>
+    </nav>
+  <ul class="offer-pop">
+  <li class="canopy">${containersworks[0].offer[0]}</li>
+  <span class="point"></span>
+  <li class="backend">${containersworks[1].offer[1]}</li>
+  <span class="point"></span>
+  <li class="year">${containersworks[2].offer[2]}</li>
+</ul>
+    <img class="snapshootpop" src="${containersworks[0].snapshoot}" alt="mobile image">
```

Comment:
- [ ] [OPTIONAL] You did a great job using a simpler method for you to implement the popup. However, this method is much of static and less dynamic. I kindly suggest looping through the array of objects and for the specific id of the object, you can show the card details. This will be more dynamic and therefore your code will be easier to understand and maintain.